clients/src/main/java/org/apache/kafka/clients/admin/ConfigEntry.java

```diff
@@ -223,6 +223,7 @@ public enum ConfigSource {
         DYNAMIC_BROKER_CONFIG,          // dynamic broker config that is configured for a specific broker
         DYNAMIC_DEFAULT_BROKER_CONFIG,  // dynamic broker config that is configured as default for all brokers in the cluster
         DYNAMIC_CLIENT_METRICS_CONFIG,  // dynamic client metrics subscription config that is configured for all clients
+        DYNAMIC_CONSUMER_GROUP_CONFIG,  // dynamic consumer group config that is configured for a specific consumer group
```

Comment:
This should be `DYNAMIC_GROUP_CONFIG` because it applies to group not just a consumer group.

Reply:
Done